### ISSUE TITLE
Provider does not import cloudflare_rate_limit.correlate

### ISSUE THREAD

**Issue by matejvelikonja** [terraform-providers/terraform-provider-cloudflare#199]
### Terraform Version
```
Terraform v0.11.11
+ provider.cloudflare v1.11.0
```

### Affected Resource(s)
- cloudflare_rate_limit

### Terraform Configuration Files
```hcl
resource "cloudflare_rate_limit" "example_com" {
  description = "Example"
  zone        = "example.com"
  threshold   = 100
  period      = 60

  match {
    request {
      url_pattern = "*"
    }
  }

  action {
    mode    = "simulate"
    timeout = 60
  }

  correlate {
    by = "nat"
  }

  disabled = false
}
```

### Expected Behavior
Terraform plan should say, that everything is up to date.

### Actual Behavior
```text
correlate.#:                   "0" => "1"
correlate.0.by:                "" => "nat"
```

### Steps to Reproduce
```bash
terraform import "cloudflare_rate_limit.example_com" "example.com/id-of-the-resource"
```

```bash
terraform plan
```

I've also checked JSON response from CloudFlare API and I received correctly:
```json
"correlate": {
    "by": "nat"
},
```

**Comment by jacobbednarz** [comment]
Thanks for reporting this one @matejvelikonja! I'm taking a look at this now.

**Comment by matejvelikonja** [comment]
@jacobbednarz I have tested it and I confirm that it has been fixed with #204
